uktrade/data-hub-frontend
# My referrals on dashboard

## Description of change

The dashboard is now a tab navigation which contains the existing _My companies_ lists and also the new _My referrals_ list, which shows all the referrals related to the current user. 

## Test instructions

1. Go to the homepage
2. You should see a tab navigation with tabs _My companies lists_ and _My referrals_
3. Click on the _My referrals_ tab
4. After the progress indicator disappears, you should either see an error message or a list of referrals
5. Click on the _My companies lists_
6. After the progress indicator disappears, you should see the original _My companies lists_ section

## Screenshots
### Before
<img width="738" alt="Screen Shot 2020-03-16 at 9 39 12 AM" src="https://user-images.githubusercontent.com/2333157/76743207-2e58fe80-676a-11ea-8cef-60dcd619a9cc.png">

### After
<img width="634" alt="Screen Shot 2020-03-16 at 9 39 42 AM" src="https://user-images.githubusercontent.com/2333157/76743230-39ac2a00-676a-11ea-8bf4-41f96a780694.png">
